Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
```

Comment:
It's not so much unmaintained as maintained by a bunch of people around the world who each work on their own fork... But after several years since the last official version (still linked to [here](http://www.cstr.ed.ac.uk/projects/speech_tools/)), and some hacks necessary to compile it on recent versions of OSX (see, e.g., [this thread on SX](http://apple.stackexchange.com/a/128979)), a new "stable" version was released and announced on 12/25 that *finally* compiles out of the box again. This is what this formula is based on.